libs/packages/layouts/src/lib/feature/subheader/subheader-drawer.component.html

```diff
@@ -0,0 +1,5 @@
+<!-- <sds-drawer (isDrawerOpen)="isDrawerOpen.emit($event)" [contentTemplate]="contentTemplate"></sds-drawer> -->
+
+<button (click)="onDrawerOpenClose($event)"  class="usa-button sds-button--circle">
+    <fa-icon [fixedWidth]="true" [icon]="isOpen ?['fas', 'arrow-down']: ['fas', 'arrow-up']"></fa-icon>
```

Comment:
Reverse the arrows:
Down arrow when closed
Up arrow when open

Reply:
reversed the icons